### PR TITLE
Refactoring of typography module

### PR DESCRIPTION
**Pull request by hopkins-tk**
All typography related elements moved to single module.

### CODE REVIEW

File: packages/typography/Code.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import "../generics.shared";
+@import "../../node_modules/blaze/scss/components.typography";
```
Comment (devopsmariocom) [inline packages/typography/Code.scss]: please use `~` instead of `../../node_modules/`

File: packages/typography/index.demo.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { h, Component } from 'skatejs';
```
Comment (devopsmariocom) [inline packages/typography/index.demo.tsx:1]: I guess we can remove demo files as well and put all demos directly here to shrink amount of files in this module... what do you think?

**Review by devopsmariocom — CHANGES_REQUESTED** [review]

File: packages/typography/index.demo.tsx (hunk 2 above)
Comment (hopkins-tk) [reply packages/typography/index.demo.tsx:1]: I have tried that at first, but it was huge file, which was not very readable, so I restored the demo files.

Comment (devopsmariocom) [reply packages/typography/index.demo.tsx:1]: yeah I meant single demo component with all examples inside, not multiple demo components within single file if that is what you mean

Comment (hopkins-tk) [reply packages/typography/index.demo.tsx:1]: Done

**Review by devopsmariocom — APPROVED** [review]

**Comment by hopkins-tk** [comment]
Closes #92

**Pull request merged**